### PR TITLE
dev-python/nautilus-python: Version Bump

### PR DESCRIPTION
**Pull request by j-licht**
and add python3 target

Bug: https://bugs.gentoo.org/673038
Package-Manager: Portage-2.3.84, Repoman-2.3.20
Signed-off-by: Jonas Licht <jonas.licht@fem.tu-ilmenau.de>

### CODE REVIEW

File: dev-python/nautilus-python/nautilus-python-1.2.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+inherit gnome2 python-single-r1
+
+DESCRIPTION="Python bindings for the Nautilus file manager"
+HOMEPAGE="https://projects.gnome.org/nautilus-python/"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm64 ~x86"
+IUSE=""
```
Comment (ConiKost) [inline dev-python/nautilus-python/nautilus-python-1.2.3.ebuild]: Since it's empty, you could drop it.

**Comment by ConiKost** [comment]
@leio I did give a try, seems to work for me with owncloud and nextcloud, but I can't test with gnome-extra/gnome-shell-extension-gsconnect. At least I can see in other linux distributions, that the package is depending on python3. Still ok to merge?

**Comment by okias** [comment]
can be merged please?

**Comment by kobboi** [comment]
Doesn't compile for me unfortunately, will log a bug on bgo

**Comment by okias** [comment]
also compile issue:
```
>>> Emerging (1 of 1) dev-python/nautilus-python-1.2.3::gentoo
 * nautilus-python-1.2.3.tar.xz BLAKE2B SHA512 size ;-) ...                                [ ok ]
 * Using python3.7 to build
>>> Unpacking source...
>>> Unpacking nautilus-python-1.2.3.tar.xz to /var/tmp/portage/dev-python/nautilus-python-1.2.3/work
>>> Source unpacked in /var/tmp/portage/dev-python/nautilus-python-1.2.3/work
>>> Preparing source in /var/tmp/portage/dev-python/nautilus-python-1.2.3/work/nautilus-python-1.2.3 ...
 * Disabling deprecation warnings ...                                                      [ ok ]
 * Running elibtoolize in: nautilus-python-1.2.3/
 *   Applying portage/1.2.0 patch ...
 *   Applying sed/1.5.6 patch ...
 *   Applying as-needed/2.4.3 patch ...
 *   Applying ppc64le/2.4.4 patch ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/dev-python/nautilus-python-1.2.3/work/nautilus-python-1.2.3 ...
 * econf: updating nautilus-python-1.2.3/config.sub with /usr/share/gnuconfig/config.sub
 * econf: updating nautilus-python-1.2.3/config.guess with /usr/share/gnuconfig/config.guess
./configure --prefix=/usr --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu --mandir=/usr/share/man --infodir=/usr/share/info --datadir=/usr/share --sysconfdir=/etc --localstatedir=/var/lib --disable-dependency-tracking --disable-silent-rules --docdir=/usr/share/doc/nautilus-python-1.2.3 --htmldir=/usr/share/doc/nautilus-python-1.2.3/html --libdir=/usr/lib64 --disable-gtk-doc
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for x86_64-pc-linux-gnu-gcc... x86_64-pc-linux-gnu-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether x86_64-pc-linux-gnu-gcc accepts -g... yes
checking for x86_64-pc-linux-gnu-gcc option to accept ISO C89... none needed
checking whether x86_64-pc-linux-gnu-gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of x86_64-pc-linux-gnu-gcc... none
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by x86_64-pc-linux-gnu-gcc... /usr/x86_64-pc-linux-gnu/bin/ld
checking if the linker (/usr/x86_64-pc-linux-gnu/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/x86_64-pc-linux-gnu-nm -B
checking the name lister (/usr/bin/x86_64-pc-linux-gnu-nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/x86_64-pc-linux-gnu/bin/ld option to reload object files... -r
checking for x86_64-pc-linux-gnu-objdump... x86_64-pc-linux-gnu-objdump
checking how to recognize dependent libraries... pass_all
checking for x86_64-pc-linux-gnu-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for x86_64-pc-linux-gnu-ar... x86_64-pc-linux-gnu-ar
checking for archiver @FILE support... @
checking for x86_64-pc-linux-gnu-strip... x86_64-pc-linux-gnu-strip
checking for x86_64-pc-linux-gnu-ranlib... x86_64-pc-linux-gnu-ranlib
checking command to parse /usr/bin/x86_64-pc-linux-gnu-nm -B output from x86_64-pc-linux-gnu-gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for x86_64-pc-linux-gnu-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking how to run the C preprocessor... x86_64-pc-linux-gnu-gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if x86_64-pc-linux-gnu-gcc supports -fno-rtti -fno-exceptions... no
checking for x86_64-pc-linux-gnu-gcc option to produce PIC... -fPIC -DPIC
checking if x86_64-pc-linux-gnu-gcc PIC flag -fPIC -DPIC works... yes
checking if x86_64-pc-linux-gnu-gcc static flag -static works... yes
checking if x86_64-pc-linux-gnu-gcc supports -c -o file.o... yes
checking if x86_64-pc-linux-gnu-gcc supports -c -o file.o... (cached) yes
checking whether the x86_64-pc-linux-gnu-gcc linker (/usr/x86_64-pc-linux-gnu/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for pkg-config... yes
checking for pkg-config... /usr/bin/pkg-config
checking for x86_64-pc-linux-gnu-pkg-config... (cached) /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for gtk-doc... yes
checking for gtkdoc-check... gtkdoc-check.test
checking for gtkdoc-check... /usr/bin/gtkdoc-check
checking for gtkdoc-rebase... /usr/bin/gtkdoc-rebase
checking for gtkdoc-mkpdf... /usr/bin/gtkdoc-mkpdf
checking whether to build gtk-doc documentation... no
checking for GTKDOC_DEPS... yes
checking for python version... 3.7
checking for python platform... linux
checking for python script directory... ${prefix}/lib/python3.7/site-packages
checking for python extension module directory... ${exec_prefix}/lib/python3.7/site-packages
checking for headers required to compile python extensions... found
checking for libraries required to embed python... yes
checking for NAUTILUS_PYTHON... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating examples/Makefile
config.status: creating src/Makefile
config.status: creating nautilus-python.pc
config.status: WARNING:  'nautilus-python.pc.in' seems to ignore the --datarootdir setting
config.status: creating docs/Makefile
config.status: creating docs/reference/entities.docbook
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands

 nautilus-python 1.2.3

    Nautilus Prefix: /usr
  PyGObject Version: pygobject-3.0
     Python Library: /usr/lib64/python3.7m
      Documentation: no

>>> Source configured.
>>> Compiling source in /var/tmp/portage/dev-python/nautilus-python-1.2.3/work/nautilus-python-1.2.3 ...
make --quiet -j4 -l6 
Making all in src
libtool: compile:  x86_64-pc-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I.. -DG_LOG_DOMAIN=\"Nautilus-Python\" -DDATADIR=\"/usr/share\" -DLIBDIR=\"/usr/lib64\" -I.. -I.. -pthread -I/usr/include/pygobject-3.0 -I/usr/include/nautilus -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/lib64/libffi/include -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -DPYTHON_VERSION=\"3.7\" -DPY_LIB_LOC=\"/usr/lib64\" -DPY_LIB_NAME=\"python3.7m\" -I/usr/include/python3.7m -I/usr/include/python3.7m -march=native -O2 -flto -fno-stack-protector -pipe -c nautilus-python.c  -fPIC -DPIC -o .libs/nautilus-python.o
libtool: compile:  x86_64-pc-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I.. -DG_LOG_DOMAIN=\"Nautilus-Python\" -DDATADIR=\"/usr/share\" -DLIBDIR=\"/usr/lib64\" -I.. -I.. -pthread -I/usr/include/pygobject-3.0 -I/usr/include/nautilus -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/lib64/libffi/include -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -DPYTHON_VERSION=\"3.7\" -DPY_LIB_LOC=\"/usr/lib64\" -DPY_LIB_NAME=\"python3.7m\" -I/usr/include/python3.7m -I/usr/include/python3.7m -march=native -O2 -flto -fno-stack-protector -pipe -c nautilus-python-object.c  -fPIC -DPIC -o .libs/nautilus-python-object.o
nautilus-python-object.c: In function ‘__PyString_AsString’:
nautilus-python-object.c:45:12: warning: return discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
   45 |     return PyUnicode_AsUTF8(obj);
      |            ^~~~~~~~~~~~~~~~~~~~~
nautilus-python-object.c: In function ‘nautilus_python_object_info_provider_iface_init’:
nautilus-python-object.c:447:29: warning: assignment to ‘NautilusOperationResult (*)(NautilusInfoProvider *, NautilusFileInfo *, GClosure *, NautilusOperationHandle **)’ {aka ‘NautilusOperationResult (*)(struct _NautilusInfoProvider *, struct _NautilusFileInfo *, struct _GClosure *, struct _NautilusOperationHandle **)’} from incompatible pointer type ‘NautilusOperationResult (*)(NautilusInfoProvider *, NautilusFile *, GClosure *, NautilusOperationHandle **)’ {aka ‘NautilusOperationResult (*)(struct _NautilusInfoProvider *, NautilusFile *, struct _GClosure *, struct _NautilusOperationHandle **)’} [-Wincompatible-pointer-types]
  447 |     iface->update_file_info = nautilus_python_object_update_file_info;
      |                             ^
libtool: link: x86_64-pc-linux-gnu-gcc -shared  -fPIC -DPIC  .libs/nautilus-python.o .libs/nautilus-python-object.o   -Wl,--as-needed -lnautilus-extension -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lpython3.7m -lcrypt -lpthread -ldl -lutil -lm  -march=native -O2 -flto -Wl,-O1   -Wl,-soname -Wl,libnautilus-python.so -o .libs/libnautilus-python.so
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyGObject_API'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusOperationHandle_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusPropertyPage_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusMenuItem_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusColumn_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyGtkWidget_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusPropertyPageProvider_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusLocationWidgetProvider_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusMenuProvider_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusColumnProvider_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusInfoProvider_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: error: .libs/nautilus-python-object.o: multiple definition of '_PyNautilusMenu_Type'
/usr/lib/gcc/x86_64-pc-linux-gnu/10.1.0/../../../../x86_64-pc-linux-gnu/bin/ld: .libs/nautilus-python.o: previous definition here
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:450: libnautilus-python.la] Error 1
```

**Pull request closed without merging**